### ISSUE TITLE
Guide to fine-tuning ReDimNet

### ISSUE THREAD

**Issue by pongthang** [IDRnD/redimnet#14]
Hi, as Redimnet was not trained in kids data, accuracy drops (60-70%) when I tried in kids dataset. So I want to fine-tune the model in kids voice. Could you help in this? Could you provide the training script or may be training script example? and some wiki how you train it. Btw it is a good project. Thanks.

**Comment by vanIvan** [comment]
Hi @pongthang , thank you for taking look at our work! We were using slightly modified version of [wespeaker](https://github.com/wenet-e2e/wespeaker) pipeline, we'll discuss in our team, when we could publish it. Foe now please refer to wespeaker pipeline, the only main difference we have - we removed feature calculation from data loading pipeline, and inserted them into the model. I believe wespeaker pipeline has added ReDimNet architecture and recipes for it.

**Comment by pongthang** [comment]
Thank you . If you could share the training process it will be great. Yeah, wespeaker pipeline has added ReDimNet architecture and recipes for it. I will follow this, thank you again for your support.

**Comment by vanIvan** [comment]
We'll be soon releasing few more models trained on much larger dataset. They should have better quality across other domains. Stay tuned.

**Comment by pongthang** [comment]
> We'll be soon releasing few more models trained on much larger dataset. They should have better quality across other domains. Stay tuned.

This is great. How can I know new models are released ?

**Comment by MonolithFoundation** [comment]
@vanIvan Will it support Chinese speaker verification? Any estimate on this? Really hoping for his.

**Comment by vanIvan** [comment]
Yes, new models would be pretrained on voxblink2 and finetuned on voxblink2+vox2+cnceleb. They will perform better on Chinese.

**Comment by pongthang** [comment]
@vanIvan , Hi , are you planning to train the new dataset on smaller models which have size similar to b0 or b1?

**Comment by MonolithFoundation** [comment]
@vanIvan sounds extremly good. Is there any estimated on why will new models release?

**Comment by vanIvan** [comment]
@MonolithFoundation we have released two new models yesterday, S and M models trained on voxblink2 dataset, please check readme and evaluation pages for more information.

**Comment by vanIvan** [comment]
@pongthang No, we were not planning to train smaller models on voxblink2 dataset.

**Comment by vanIvan** [comment]
@MonolithFoundation we have already released them yesterday, both models S and M have two sets of weights: pretrained version on voxblink2 and finetuned version on voxblink2+vox2+cnceleb12

**Comment by MonolithFoundation** [comment]
Thank you so much.
Is there any inference script that can be used to infer on these models?

**Comment by rose-jinyang** [comment]
Hi @vanIvan 
When can you release your training code?

**Comment by vanIvan** [comment]
Hello @Shuo-H , it would take around 2 weeks on 2 x A100 (80GB per GPU) for pretraining and 3-4 days to fine-tune

**Comment by vanIvan** [comment]
@rose-jinyang sorry for not answering for so long, we don't have time currently to release training code. Also as we stated in our paper, we used [wespeaker](https://github.com/wenet-e2e/wespeaker) pipeline with few minor changes, that are not critical to models performance. You should be able to use wespeaker pipeline almost as is.
